examples/nemo/nemo_gpt_distributed.yaml

```diff
@@ -0,0 +1,139 @@
+# Distributed training a GPT style model with Nvidia NeMo on multiple nodes.
+#
+# Inspired from https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/nemo_megatron/gpt/gpt_training.rst
+#
+# Note that we provide a read-only bucket at gs://sky-wiki-data that is used to
+# download preprocessed data to local disk. If you want to preprocess the data
+# yourself, see nemo_gpt_preprocessing.yaml.
+#
+# We use a shared bucket to store the index files that are used to coordinate
+# between the head and worker nodes. This shared bucket is mounted as a
+# network filesystem (NFS) on the head and worker nodes.
+#
+# After the script completes, the model checkpoints will be saved in
+# /ckpts on the head node (can be changed to /shared for cloud storage).
+#
+# Usage:
+#   sky launch --env SHARED_NFS_BUCKET_NAME=<unique_bucket_name> -c nemo_gpt nemo_gpt_distributed.yaml
+#
+#   # Terminate cluster after you're done
+#   sky down nemo_gpt
+
+resources:
+  cpus: 8+
+  memory: 64+
+  accelerators: A100-80GB:1
```

Comment:
Should we have a test with multi-node multi-gpu as well, i.e. each node having multiple GPUs?

Reply:
Confirmed it works with `A100-80GB:4` on 2 nodes!